tf-modules/archive/api.tf

```
@@ -0,0 +1,117 @@
+locals {
+  api_port_substring = var.api_port == null ? "" : ":${var.api_port}"
+  api_uri            = var.api_url == null ? "https://${aws_api_gateway_rest_api.api.id}.execute-api.${data.aws_region.current.name}.amazonaws.com${local.api_port_substring}/${var.api_gateway_stage}/" : var.api_url
+  api_redirect_uri   = "${local.api_uri}token"
+}
+
+resource "aws_cloudwatch_log_group" "api" {
+  name              = "/aws/lambda/${aws_lambda_function.api.function_name}"
+  retention_in_days = 30
+}
+
+resource "aws_lambda_function" "api" {
+  depends_on = [aws_iam_role.lambda_api_gateway]
+
+  function_name    = "${var.prefix}-ApiEndpoints"
+  filename         = "${path.module}/../../packages/api/dist/app/lambda.zip"
+  source_code_hash = filebase64sha256("${path.module}/../../packages/api/dist/app/lambda.zip")
+  handler          = "index.handler"
+  role             = aws_iam_role.lambda_api_gateway.arn
+  runtime          = "nodejs8.10"
+  timeout          = 20
```

```suggestion
  timeout          =  100
```
The current configuration timeout is @jennyhliu  updated it in 8d56b9e